src/github/tests/test_utils.rs

```diff
@@ -92,12 +92,29 @@ impl DataModel {
         GithubMock {
             users,
             owners: Default::default(),
+            members: Default::default(),
             teams,
             team_memberships,
             team_invitations: Default::default(),
         }
     }
 
+    pub fn diff_toml_gh_org_teams(
+        &self,
+        github: GithubMock,
+        // members: HashSet<u64>,
+    ) -> OrgMembershipDiff {
+        let teams: Vec<TomlTeam> = self.teams.iter().map(|r| r.to_data()).collect();
+        let repos = vec![];
+        let read = Box::new(github);
+        let sync = SyncGitHub::new(read, teams, repos).expect("Cannot create SyncGitHub");
+
+        let org_team_members = sync.map_teams_to_org().unwrap();
+
+        sync.diff_teams_gh_org(org_team_members)
```

Comment:
This test won't work yet, because our test API is not finished yet and this uses the unfinished parts. We need to properly track GH users and GH org members in `GithubMock`, which doesn't happen currently :( I will work on it and send a PR so that you can write the test and check it locally